src/strategies/rdnt-capital-locked-dlp/index.ts

```diff
@@ -0,0 +1,71 @@
+import { Multicaller } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+import { BigNumber } from '@ethersproject/bignumber';
+import fetch from 'cross-fetch';
+
+export const author = 'JDoy99';
+export const version = '0.1.0';
+
+const abi = [
+  {
+    name: 'lockedUsdValue',
+    type: 'function',
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'user',
+        type: 'address'
+      }
+    ],
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view'
+  }
+];
+
+const contractAddress = '0xd4966DC49a10aa5467D65f4fA4b1449b5d874399';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+
+  addresses.forEach((address) => {
+    multi.call(`${address}.lockedUsdValue`, contractAddress, 'lockedUsdValue', [
+      address
+    ]);
+  });
+
+  async function getTokenPrice() {
+    const url = `https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=radiant-capital`;
+    const response = await fetch(url);
+    const data = await response.json();
+    return data[0].current_price;
```

Comment:
The price should be depended on `snapshot or its timestamp, else voting power will not be same as expected everytime, and can cause lot of problems, refer to this implementation https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/digitalax-genesis-contribution/index.ts#L137-L139
It should calculate voting power based on timestamp